packages/itinerary-body/src/ItineraryBody/index.js

```diff
@@ -212,12 +215,24 @@ ItineraryBody.propTypes = {
    * A custom icon component inserted into the transit alert body component
    * within a transit leg, if this prop is not supplied a default icon is used
    */
-  AlertBodyIcon: PropTypes.elementType
+  AlertBodyIcon: PropTypes.elementType,
+  /**
+   * A mapping of accessibility score to color, icon, and text used
+   * to override the default one shipped in AccessibilityLabel
+   */
+  accessibilityScoreGradationMap: PropTypes.shape({
+    color: PropTypes.string,
+    icon: PropTypes.element,
+    text: PropTypes.string
+  })
```

Comment:
Sort props.

Reply:
Extract this shape into a var, export, and reuse in this package. In light of TypeScript-ing this component in some near future, I would skip adding the var to `core-utils/types`.

Reply:
See previous comment on this